### DISCUSSION TITLE
Complejidad temporal solución parte 2

### DISCUSSION THREAD

**Discussion by k0rval4n** [IIC2133-PUC/2024-2#188] [Tarea 3] [answered]
¡Hola! En el punto 2.3 de la tarea se solicita la complejidad temporal de la solución. ¿Debemos incluir en este cálculo el iterar sobre los L inputs iniciales que nos entregan, o solo por lo que se hace después de tener cargados a los N maestros y L posibles sesiones de entrenamiento?

¡Muchas gracias!

**Comment by Segismundios** [comment, marked as answer]
Holaa @k0rval4n. Puedes incluirlo siempre y cuando especifiques de dónde viene esa complejidad. Igualmente, recuerda que al usar notación asintótica nos enfocamos en el término de mayor crecimiento.
Saludos!

**Reply by k0rval4n** [reply]
Super, muchas gracias! 🚀
